### PR TITLE
refactor: created filters package

### LINKED ISSUE #155: (1) [filters] Merge @dfwc/terms and @dfwc/range into @dfwc/filters

### PR DESCRIPTION
**Pull request by hailem**
fixes #155

### CODE REVIEW

File: playground/demo/classic.html [hunk 1]
```diff
@@ -112,14 +112,7 @@
...
   <script type="module">
-    import '@dfwc/results';
-    import '@dfwc/input';
-    import '@dfwc/layer';
-    import '@dfwc/panel';
-    import '@dfwc/range';
-    import '@dfwc/suggestions';
-    import '@dfwc/terms';
-    import '@dfwc/total';
+    import '@dfwc/webcomponents';
```
Comment (carlosescri) [inline playground/demo/classic.html]: Don't do this in the playgrounds for the moment.

File: playground/demo/compact.html [hunk 2]
```diff
@@ -31,14 +31,7 @@
...
   <script type="module">
-    import '@dfwc/results';
-    import '@dfwc/input';
-    import '@dfwc/display';
-    import '@dfwc/layer';
-    import '@dfwc/panel';
-    import '@dfwc/range';
-    import '@dfwc/suggestions';
-    import '@dfwc/terms';
+    import '@dfwc/webcomponents';
```
Comment (carlosescri) [inline playground/demo/compact.html]: The same here…

File: playground/demo/embedded.html [hunk 3]
```diff
@@ -66,15 +66,7 @@
...
-    import '@dfwc/results';
-    import '@dfwc/input';
-    import '@dfwc/layer';
-    import '@dfwc/pager';
-    import '@dfwc/panel';
-    import '@dfwc/range';
-    import '@dfwc/suggestions';
-    import '@dfwc/terms';
-    import '@dfwc/total';
+    import '@dfwc/webcomponents';
```
Comment (carlosescri) [inline playground/demo/embedded.html]: And here…

File: playground/demo/fullscreen.html [hunk 4]
```diff
@@ -83,14 +83,7 @@
...
   <script type="module">
-    import '@dfwc/results';
-    import '@dfwc/input';
-    import '@dfwc/layer';
-    import '@dfwc/panel';
-    import '@dfwc/range';
-    import '@dfwc/suggestions';
-    import '@dfwc/terms';
-    import '@dfwc/total';
+    import '@dfwc/webcomponents';
```
Comment (carlosescri) [inline playground/demo/fullscreen.html]: And here…

File: playground/demo/massimo-dutti.html [hunk 5]
```diff
@@ -64,14 +64,7 @@
...
   <script type="module">
-    import '@dfwc/history';
-    import '@dfwc/input';
-    import '@dfwc/layer';
-    import '@dfwc/panel';
-    import '@dfwc/results';
-    import '@dfwc/sort';
-    import '@dfwc/suggestions';
-    import '@dfwc/terms';
+    import '@dfwc/webcomponents';
```
Comment (carlosescri) [inline playground/demo/massimo-dutti.html]: And here…

File: playground/demo/classic.html (hunk 1 above)
Comment (hailem) [reply playground/demo/classic.html]: done

File: playground/demo/compact.html (hunk 2 above)
Comment (hailem) [reply playground/demo/compact.html]: done

File: playground/demo/embedded.html (hunk 3 above)
Comment (hailem) [reply playground/demo/embedded.html]: done

File: playground/demo/fullscreen.html (hunk 4 above)
Comment (hailem) [reply playground/demo/fullscreen.html]: done

File: playground/demo/massimo-dutti.html (hunk 5 above)
Comment (hailem) [reply playground/demo/massimo-dutti.html]: done

**Pull request merged**